docs/design/features/offload_XDP/design_and_constraints.md

```diff
@@ -0,0 +1,32 @@
+<!--
+SPDX-License-Identifier: MIT
+Copyright (c) 2021 The Authors.
+
+Authors:
+Peng Yang               (@yangpenger)      
+
+-->
+
+Considering the limitation of the SmartNIC (cannot redirect, and only supports HASH and ARRAY type map), bouncers and dividers (forwarding function) are offloaded to the NIC. 
```

Comment:
Could we please bullet-point the limitations. 
e.g. Netronome NIC (model NNN) has the following limitations:
- Does not support XDP_REDIRECT
- Does not support tail calls.
- ...